lib/ansiblelint/utils.py

```diff
@@ -364,18 +364,32 @@ def _kv_to_dict(v):
 def normalize_task_v2(task):
     """Ensure tasks have an action key and strings are converted to python objects."""
     result = dict()
-    mod_arg_parser = ModuleArgsParser(task)
+    clean_task = task.copy()
```

Comment:
Mind moving the copy after the if-block too? There's no use for it when it enters the raise-branch. So why waste memory?